### PR TITLE
Test auth new user

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @MichaelButkovic 

Adding this so we can let combine talk to our sites willy-nilly.

### CODE REVIEW

File: tests/api/test_token_auth.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APIClient
+
+from bulbs.utils.test import BaseIndexableTestCase
+
+
+User = get_user_model()
+
+
+class TokenAuthTestCase(BaseIndexableTestCase):
```
Comment (mparent61) [inline tests/api/test_token_auth.py]: Nice coverage, but I'd also add test that fails auth

File: bulbs/api/management/commands/create_app_authtoken.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+User = get_user_model()
+
+
+class Command(BaseCommand):
+
+    help = "Creates a new user & app token for a client request."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-username',
```
Comment (mparent61) [inline bulbs/api/management/commands/create_app_authtoken.py]: Nit: needs two hypens "--username" for long args

File: bulbs/api/management/commands/create_app_authtoken.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            dest='username',
+            help='The username for the given application.',
+            required=True,
+            type=str
+        )
+
+    def handle(self, *args, **kwargs):
+        username = kwargs.get('username')
+        qs = User.objects.filter(username=username)
```
Comment (mparent61) [inline bulbs/api/management/commands/create_app_authtoken.py]: get_or_create?

Comment (benghaziboy) [inline bulbs/api/management/commands/create_app_authtoken.py]: I think the problem with get_or_create & users is password requirement

Comment (mparent61) [inline bulbs/api/management/commands/create_app_authtoken.py]: Oh? Not solved by defaults={} ?

File: bulbs/api/management/commands/create_app_authtoken.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        if not qs.exists():
+            user = User.objects.create(
+                username=username,
+                is_staff=True,
+                password=random.getrandbits(128)
```
Comment (mparent61) [inline bulbs/api/management/commands/create_app_authtoken.py]: random library isn't cryptographically secure, better to use something like os.urandom or SystemRandom

http://stackoverflow.com/questions/20936993/how-can-i-create-a-random-number-that-is-cryptographically-secure-in-python

**Comment by mparent61** [comment]
Useful command!

**Comment by MichaelButkovic** [comment]
LGTM

**Pull request merged**